test/e2e/config/docker.yaml

```
@@ -199,10 +199,10 @@ variables:
   # allowing the same e2e config file to be re-used in different Prow jobs e.g. each one with a K8s version permutation.
   # The following Kubernetes versions should be the latest versions with already published kindest/node images.
   # This avoids building node images in the default case which improves the test duration significantly.
-  KUBERNETES_VERSION_MANAGEMENT: "v1.23.0"
-  KUBERNETES_VERSION: "v1.23.0"
+  KUBERNETES_VERSION_MANAGEMENT: "v1.23.3"
+  KUBERNETES_VERSION: "v1.23.3"
   KUBERNETES_VERSION_UPGRADE_FROM: "v1.22.4"
```

What about picking up also latest from v1.22 given that we are at it...
```suggestion
  KUBERNETES_VERSION_UPGRADE_FROM: "v1.22.6"
```

---

@sbueringer ^^

---

If I see correctly there is no published image for v1.22.6 (https://hub.docker.com/r/kindest/node/tags) which means the job which uses that version has to build v1.22.6 every time.

---

fair point, let's hold on this